### PR TITLE
tests: import tests with details from snapd-testing-tools

### PR DESCRIPTION
**Pull request by sergiocazzolato**
Include the new tests with details from snapd-testing-tools

### CODE REVIEW

**Comment by sergiocazzolato** [comment]
skiped spread tests because no tools updated, just details added to task.yaml files

File: tests/lib/external/snapd-testing-tools/tests/log-analyzer/task.yaml [hunk 1]
```diff
@@ -1,5 +1,10 @@
 summary: integration tests for log-analyzer tool
 
+details: |
+    This test checks the log-analyzer retrieves properly the tests-to-reexecute
+    based on the spread log analysis. Also checks it retrieves properly the
+    tests by result status and all the tasks executed.
```
Comment (MiguelPires) [inline tests/lib/external/snapd-testing-tools/tests/log-analyzer/task.yaml:6]: nitpick: minor rewording 
```suggestion
    This test checks the log-analyzer properly retrieves the tests-to-reexecute
    based on the spread log analysis. Also checks it properly retrieves the
    tests by result status and all the executed tasks.
```

File: tests/lib/external/snapd-testing-tools/tests/os.paths/task.yaml [hunk 2]
```diff
@@ -1,5 +1,9 @@
 summary: smoke test for the os.paths tool
 
+details: |
+    Check the os.paths tool retrieves properly the snap-mount-dir,
```
Comment (MiguelPires) [inline tests/lib/external/snapd-testing-tools/tests/os.paths/task.yaml:4]:
```suggestion
    Check the os.paths tool properly retrieves the snap-mount-dir,
```

File: tests/lib/external/snapd-testing-tools/tests/os.query/task.yaml [hunk 3]
```diff
@@ -1,5 +1,9 @@
 summary: smoke test for the os.query tool
 
+details: |
+    Check the os.query tool detects properly the current system. Also
+    verify the versions comparison works.
```
Comment (MiguelPires) [inline tests/lib/external/snapd-testing-tools/tests/os.query/task.yaml:5]:
```suggestion
    Check the os.query tool properly detects the current system. Also
    verify the version comparison works.
```

File: tests/lib/external/snapd-testing-tools/tests/not/task.yaml [hunk 4]
```diff
@@ -1,5 +1,8 @@
 summary: test for the not tool
 
+details: |
+    Check the not tool works the same than the logical not (!)
```
Comment (MiguelPires) [inline tests/lib/external/snapd-testing-tools/tests/not/task.yaml:4]:
```suggestion
    Check the not tool works the same as the logical not (!).
```

File: tests/lib/external/snapd-testing-tools/tests/remote.setup/task.yaml [hunk 5]
```diff
@@ -1,5 +1,10 @@
 summary: smoke test for the remote.setup tool
 
+details: |
+    Check the remote.setup tool sets up up properly the
+    information required to allow connecting to the remote
+    instance through ssh
```
Comment (MiguelPires) [inline tests/lib/external/snapd-testing-tools/tests/remote.setup/task.yaml:6]:
```suggestion
    Check the remote.setup tool properly sets up up the
    information required to allow connecting to the remote
    instance through ssh.
```

File: tests/lib/external/snapd-testing-tools/tests/retry/task.yaml [hunk 6]
```diff
@@ -1,5 +1,9 @@
 summary: smoke test for the retry tool
 
+details: |
+    Check the retry tool allows to retry a command. Verify that on
```
Comment (MiguelPires) [inline tests/lib/external/snapd-testing-tools/tests/retry/task.yaml:4]:
```suggestion
    Check the retry tool allows retrying a command. Verify that on
```

File: tests/lib/external/snapd-testing-tools/tests/snaps.name/task.yaml [hunk 7]
```diff
@@ -1,5 +1,10 @@
 summary: smoke test for the snaps.name tool
 
+detrails: |
+    Check the snaps.name tool retrieves properly the names of the
```
Comment (MiguelPires) [inline tests/lib/external/snapd-testing-tools/tests/snaps.name/task.yaml:4]:
```suggestion
    Check the snaps.name tool properly retrieves the names of the
```

File: tests/lib/external/snapd-testing-tools/tests/tests.pkgs/task.yaml [hunk 8]
```diff
@@ -1,5 +1,10 @@
 summary: Smoke test for tests.pkgs tool
 
+details: |
+    Check the tests.pkgs tool allows to manage native packages independently
```
Comment (MiguelPires) [inline tests/lib/external/snapd-testing-tools/tests/tests.pkgs/task.yaml:4]:
```suggestion
    Check the tests.pkgs tool allows managing native packages independently
```

File: tests/lib/external/snapd-testing-tools/tests/tests.systemd/task.yaml [hunk 9]
```diff
@@ -1,5 +1,10 @@
 summary: smoke test for the tests.systemd tool
 
+details: |
+    Check the tests.systemd manages properly systemd services.
```
Comment (MiguelPires) [inline tests/lib/external/snapd-testing-tools/tests/tests.systemd/task.yaml:4]:
```suggestion
    Check the tests.systemd properly manages systemd services.
```

File: tests/lib/external/snapd-testing-tools/tests/spread-manager/task.yaml [hunk 10]
```diff
@@ -1,5 +1,9 @@
 summary: smoke test for the spread-manager tool
 
+details: |
+    Check the spread-manager tool allows to set a list of tests as manual.
```
Comment (MiguelPires) [inline tests/lib/external/snapd-testing-tools/tests/spread-manager/task.yaml:4]:
```suggestion
    Check the spread-manager tool allows setting a list of tests as manual.
```

File: tests/lib/external/snapd-testing-tools/tests/spread-shellcheck/task.yaml [hunk 11]
```diff
@@ -1,5 +1,11 @@
 summary: smoke test for the spread-shellcheck tool
 
+details: |
+    Check the spread-shellcheck properly runs the shellcheck
+    tool in all the sections of the spread tasks. Also check the
+    tool allows to run in a directory tree and skips checks when
```
Comment (MiguelPires) [inline tests/lib/external/snapd-testing-tools/tests/spread-shellcheck/task.yaml:6]:
```suggestion
    tool can run in a directory tree and skips checks when
```

**Pull request merged**